airbyte-db/db-lib/src/main/java/io/airbyte/db/PostgresUtils.java

```diff
@@ -23,4 +27,46 @@ public static PgLsn getLsn(final JdbcDatabase database) throws SQLException {
     return PgLsn.fromPgString(jsonNodes.get(0).get("pg_current_wal_lsn").asText());
   }
 
+  @VisibleForTesting
+  public static Triple<String, String, String> getCertificate(PostgreSQLContainer<?> container) throws IOException, InterruptedException {
+    container.execInContainer("su", "-c", "psql -U test -c \"CREATE USER postgres WITH PASSWORD 'postgres';\"");
```

Comment:
Sorry, but for some reason my IDEA can't accept this as text. Instead, I get the following:
![2022-07-03_19-54](https://user-images.githubusercontent.com/88329385/177049688-b1384424-ed9c-42c0-8ea7-96cc3b0ce1e4.png)

As for combining everything into one command, it won't work, because each command must be executed with SUDO rights, which I do at the beginning of each command. Also, if I do that, it will get mixed up and look unreadable. So it's best to leave it split into separate commands.

Reply:
If you open the project structure settings (`cmd`-`;` on macos, not sure on windows) you can set the language level to 17:

![Screen Shot 2022-07-05 at 3 36 37 PM](https://user-images.githubusercontent.com/5741425/177428659-407ce32a-676b-4a0d-855a-830d03982624.png)

which should enable multiline strings and record classes.